lib/tags/note.js

```diff
@@ -41,6 +41,8 @@ module.exports = function note (hexo, args, content) {
 
   return Promise.resolve(
     toMarkdown(content)
+    .replace('–', '--')
+    .replace('—', '---')
```

Comment:
Maybe let's actually add a comment in the code here about how this is a temp fix and maybe link to this issue. I can see our future selves looking at this code like 😳 wat...

Reply:
😆 

Yes, good suggestion Jennifer. I have done so.